io/plugins/eu.esdihumboldt.hale.io.validation/plugin.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="eu.esdihumboldt.hale.io.action">
+      <action
+            categoryName="Validator configurations"
+            id="eu.esdihumboldt.hale.io.validation.read"
+            name="Load validator configuration"
+            resourceName="validator configuration"
+            type="eu.esdihumboldt.hale.io.validation.ValidationConfigurationReaderImpl">
+      </action>
+      <advisor
+            action="eu.esdihumboldt.hale.io.validation.read"
+            class="eu.esdihumboldt.hale.io.validation.ValidatorConfigurationImportAdvisor"
+            id="eu.esdihumboldt.hale.io.validation.service">
+      </advisor>
+   </extension>
+   <extension
+         point="eu.esdihumboldt.util.resource">
+      <resourceType
+            id="validatorConfiguration"
+            name="Validator Configuration">
+      </resourceType>
+   </extension>
+   <extension
+         point="eu.esdihumboldt.hale.io.provider">
+      <provider
+            class="eu.esdihumboldt.hale.io.validation.ValidationConfigurationReaderImpl"
+            id="eu.esdihumboldt.hale.io.validation.reader"
+            name="Validator configuration">
+         <contentType
+               ref="eu.esdihumboldt.hale.io.schematron.schema">
+         </contentType>
+      </provider>
+      <provider
+            class="eu.esdihumboldt.hale.io.validation.ProjectValidator"
+            id="eu.esdihumboldt.hale.io.validation.projectvalidator"
+            name="Project validator">
+         <contentType
```

Comment:
The content type references here raise an interesting point. Right now it would be OK like that, but the validator should actually state that it can perform validation on any content type supported by a validator it delegates to.